### ISSUE TITLE
macOS - mount error: cannot locate OSXFUSE

### ISSUE THREAD

**Issue by panz3r** [kopia/kopia#811]
Hi, 

When trying to mount all snapshots as per the doc on macOS Big Sur
```sh
kopia mount all ./tmp/kmnt 
```

I get the following error
```sh
kopia: error: mount error: error creating fuse connection: cannot locate OSXFUSE, try --help
```

`macFuse` is correctly installed and works with other tools.

What can I do? 
Is there a way to mount the snapshots using webDAV (as stated on #5 ) instead?

### Environment

**OS**: `macOS Big Sur (11.1)`
**kopia**: `CLI 0.7.3 build: c3ade4b0ea7a1d20497fcfc4f4aec161f73c9505`
**macFuse**: `4.0.5`

**Comment by jkowalski** [comment]
It seems that something must have changed in macfuse 4.x, version 3.x worked fine, but when I upgraded I started getting the same thing...

**Comment by rjkroege** [comment]
Like @panz3r, I was puzzled by this. For reference:

* MacFuse changed it's API surface [https://github.com/osxfuse/osxfuse/releases/tag/macfuse-4.0.0](https://github.com/osxfuse/osxfuse/releases/tag/macfuse-4.0.0)
* Only MacFuse 4.0.0 and up runs on Apple Silicon
* `bazil.org/fuse` no longer supports MacFuse [commit](https://github.com/bazil/fuse/commit/60eaf8f021ce00e5c52529cdcba1067e13c1c2c2)

At least on Intel Macs, probably the easiest solution is not to upgrade past macfuse 3.* and pin `bazil.org/fuse`?

**Comment by panz3r** [comment]
Hi @rjkroege ,

Thanks for your clarifications, I think that this issue can then be linked to issue #453 .

**Comment by rjkroege** [comment]
@jkowalski #825 is a possible short term workaround for this issue.

**Comment by jkowalski** [comment]
Fixed with #829

**Comment by panz3r** [comment]
Hi @jkowalski ,
Thanks for the fix, you've been really quick 👍 

Any ETA for the next release containing this fix?
